pcl_localization:
    ros__parameters:
      registration_method: "NDT"
      ndt_resolution: 3.0
      ndt_step_size: 0.1
      transform_epsilon: 0.01
      voxel_leaf_size: 1.0
      scan_max_range: 100.0
      scan_min_range: 1.0
      scan_period: 0.1
      use_pcd_map: true
      map_path:  "/home/wheitman/navigator/data/maps/grand_loop/grand_loop.pcd"
      set_initial_pose: true
      initial_pose_x: -167.2765694799405
      initial_pose_y: 95.06041088791315
      initial_pose_z: -0.0029023253481597067
      initial_pose_qx: 0.0
      initial_pose_qy: 0.0
      initial_pose_qz: 0.6855524536695163
      initial_pose_qw: 0.7280232367635704
      use_odom: true
      use_imu: true
      enable_debug: true